tests/common/plugins/conditional_mark/__init__.py

```diff
@@ -174,8 +199,30 @@ def load_basic_facts(session):
     """
     results = {}
 
+    testbed_name = session.config.option.testbed
+    testbed_file = session.config.option.testbed_file
+
+    testbed_module = imp.load_source('testbed', 'common/testbed.py')
+    tbinfo = testbed_module.TestbedInfo(testbed_file).testbed_topo.get(testbed_name, None)
+
+    results['topo_type'] = tbinfo['topo']['type']
+    results['topo_name'] = tbinfo['topo']['name']
+
+    dut_name = tbinfo['duts'][0]
+    if session.config.option.customize_inventory_file:
+        inv_name = session.config.option.customize_inventory_file
+    elif 'inv_name' in tbinfo.keys():
+        inv_name = tbinfo['inv_name']
+    else:
+        inv_name = 'lab'
+
+    # Load DUT basic facts
+    _facts = load_dut_basic_facts(session, inv_name, dut_name)
+    if _facts:
+        results.update(_facts)
+
     # Load DUT basic facts
-    _facts = load_dut_basic_facts(session)
+    _facts = load_feature_status(session, inv_name, dut_name)
```

Comment:
The feature status is also basic DUT facts. Why not enhance the `dut_basic_facts` module to include the feature status?
Then one round of ansible-playbook session is required. Using two ansible-playbook sessions here need more overhead.

Reply:
done